### ISSUE TITLE
(during writeGitIgnore) ENOENT: no such file or directory

### ISSUE THREAD

**Issue by klebermolica** [oracle/ojet-cli#6]
I'm using Windows 7 SP1 and tried to create a web app using ojet-cli but without success because an error when creating .gitignore file

C:\Users\Administrator>npm -v
6.10.2

C:\Users\Administrator>npm -ls list -g --depth=0

| C:\Users\Administrator\AppData\Roaming\npm
|
+-- @oracle/ojet-cli@7.1.0
|   Oracle JET Command Line Interface
|   http://oraclejet.org
+-- bower@1.8.8
|   The browser package manager
|   git+https://github.com/bower/bower.git
|   http://bower.io
+-- generator-oraclejet@3.2.0
|   Yeoman generator for Oracle JET web and mobile applications
|   http://oraclejet.org
+-- grunt-cli@1.3.2
|   The grunt command line interface
|   git+https://github.com/gruntjs/grunt-cli.git
|   https://github.com/gruntjs/grunt-cli#readme
+-- node-pre-gyp@0.13.0
|   Node.js native addon binary install tool
|   git://github.com/mapbox/node-pre-gyp.git
|   https://github.com/mapbox/node-pre-gyp#readme
+-- npm@6.10.2
|   a package manager for JavaScript
|   git+https://github.com/npm/cli.git
|   https://docs.npmjs.com/
`-- yo@3.1.0
    CLI tool for running Yeoman generators
    git+https://github.com/yeoman/yo.git
    http://yeoman.io

C:\Users\Administrator>ojet create web-app-navbar --template=navbar
{
  errno: -4058,
  code: 'ENOENT',
  syscall: 'link',
  path: 'C:\\Users\\Administrator\\web-app-navbar\\_gitignore',
  dest: 'C:\\Users\\Administrator\\web-app-navbar\\.gitignore',
  message: "(during writeGitIgnore) ENOENT: no such file or directory, link 'C:\
\Users\\Administrator\\web-app-navbar\\_gitignore' -> 'C:\\Users\\Administrator\
\web-app-navbar\\.gitignore'"
}

**Comment by klebermolica** [comment]
Found out the error is being pushed from 
C:\Users\Administrator\AppData\Roaming\npm\node_modules\@oracle\ojet-cli\common\index.js
ln 15 col 42

```
module.exports =
{
  writeGitIgnore: function _writeGitIgnore() {
    **const gitSource = path.resolve('.', '_gitignore');**
    const gitDest = path.resolve('.', '.gitignore');

    return new Promise((resolve, reject) => {
      fs.move(gitSource, gitDest, (err) => {
        if (err) {
          reject(commonMessages.error(err, 'writeGitIgnore'));
        } else {
          resolve();
        }
      });
    });
  },
```

I renamed the filename of
**C:\Users\Administrator\AppData\Roaming\npm\node_modules\@oracle\ojet-cli\template\common\\_gitignore**
to
**C:\Users\Administrator\AppData\Roaming\npm\node_modules\@oracle\ojet-cli\template\common\\.gitignore**

I commented the ln 55 of
C:\Users\Administrator\AppData\Roaming\npm\node_modules\@oracle\ojet-cli\generators\app\index.js

`.then(() => common.writeGitIgnore())
`
ran again the command 

C:\Users\Administrator>ojet create web-app-navbar --template=navbar
Processing template: navbar
Your app structure is generated. Continuing with library install.
Writing:
oraclejetconfig.json
oraclejetconfig.json file exists. Checking config.
Running after_app_create hook.
Your app is ready! Change to your new app directory 'web-app-navbar' and try 'oj
et build' and 'ojet serve'.
C:\Users\Administrator\web-app-navbar>ojet build
Warning: Command is missing platform. Default to web.
Cleaning staging path.
Running before_build hook.
Copy files to staging directory.
Copy finished.
Copy library files to staging directory.
Copy finished.
Copy reference components to staging directory.
Copy finished.
Optimizing svg into SVG sprites.
Svg optimisation task finished.
Compiling sass.
Sass compile finished.
Task index.html cdn bundle injection finished.
Running theme injection task.
Task index.html theme path injection finished.
Running theme copy task.
Theme copy task finished.
Running injection tasks.
Task main.js paths injection finished.
runAllComponentHooks
Running after_build hook.
Build finished.

C:\Users\Administrator\web-app-navbar>ojet serve
Warning: Command is missing platform. Default to web.
Build: true
BuildType: dev
Destination: undefined
Destination target: undefined
Livereload: true
Livereload port: 35729
Platform: web
Port: 8000
Theme: alta
Theme platform: web
Theme version: 7.1.0
Building app.
Cleaning staging path.
Running before_build hook.
Copy files to staging directory.
Copy finished.
Copy library files to staging directory.
Copy finished.
Copy reference components to staging directory.
Copy finished.
Optimizing svg into SVG sprites.
Svg optimisation task finished.
Compiling sass.
Sass compile finished.
Task index.html cdn bundle injection finished.
Running theme injection task.
Task index.html theme path injection finished.
Running theme copy task.
Theme copy task finished.
Running injection tasks.
Task main.js paths injection finished.
runAllComponentHooks
Running after_build hook.
Running before_serve hook.
Starting web server.
Connecting to http://localhost:8000
Starting watcher.
Running after_serve hook.
Server ready: http://localhost:8000
Listening on port 35729.
Watching files.
